aws-kendra-faq/src/test/java/software/amazon/kendra/faq/CreateHandlerTest.java

```diff
@@ -74,6 +74,7 @@ public void handleRequest_SimpleSuccess() {
         String name = "name";
         String s3Key = "s3Key";
         String s3Bucket = "s3Bucket";
+        String fileFormat = "CSV";
```

Comment:
As fileFormat is optional do we have a test for that case?

Reply:
Added new test.